src/main/java/school/lemon/changerequest/java/vehicles/Airplane.java

```diff
@@ -0,0 +1,90 @@
+package school.lemon.changerequest.java.vehicles;
+
+public class Airplane extends Vehicle implements IAirplane {
+
+    private int maximumHeightFeet;
+
+    Airplane(int manufacturedYear, String make, String model, int maximumHeightFeet) {
+        super(manufacturedYear, make, model);
+        this.maximumHeightFeet = maximumHeightFeet;
+    }
+
+    /**
+     * Get airplane maximum height
+     *
+     * @return airplane maximum height
+     */
+    public int getMaximumHeightFeet() {
+        return maximumHeightFeet;
+    }
+
+    /**
+     * Set airplane maximum height
+     *
+     * @param maximumHeightFeet maximum height of the airplane
+     */
+    public void setMaximumHeightFeet(int maximumHeightFeet) {
+        this.maximumHeightFeet = maximumHeightFeet;
+    }
+
+    /**
+     * @return return string representation in the following formats
+     * for cars: This car is a 1999 Toyota Corrola with 140 hp.
```

Comment:
While you're creating an `Airplane`, do you need information about `cars` and `boats` here? Comments are not always good...